dist/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Do list</title>
+  <link rel="preconnect" href="https://fonts.gstatic.com" />
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet" />
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet" />
+  <script src="https://kit.fontawesome.com/b7a7cffe76.js" crossorigin="anonymous"></script> 
+<script defer src="/main.js"></script></head>
+
+<body>
+  <p class="title">To-Do list</p>
+
+  <div id="toDoList">
+    <div class="addTask">
+      <i class="fa fa-plus-circle complete"></i>
+      <input type="text" id="input" placeholder="Add to your list...">
+    </div>
+
+    <ul id="list">
+    </ul>
+
+    <button class="clear" onclick="clearList()">Clear list</button>

```

Comment:
- [ ] As seen in the design, the element `Clear list` should be `Clear all completed`.
- [ ] Also replace the `delete icon` with the `three dot icon`.

| Original | Yours |
| ------- | ---- |
| <img src='https://user-images.githubusercontent.com/33323077/126761942-83ad71af-f7ab-4263-8938-026f22b54043.png' width='250px'> | <img src='https://user-images.githubusercontent.com/33323077/126762007-10e5e405-eee9-4c6a-8957-fc40a63254e6.png' width='250px'> |